libs/remix-ui/workspace/src/lib/actions/workspace.ts

```diff
@@ -208,6 +208,7 @@ export const createWorkspace = async (
     if (workspaceTemplateName === 'semaphore' || workspaceTemplateName === 'hashchecker' || workspaceTemplateName === 'rln') {
       const isCircomActive = await plugin.call('manager', 'isActive', 'circuit-compiler')
       if (!isCircomActive) await plugin.call('manager', 'activatePlugin', 'circuit-compiler')
+      _paq.push(['trackEvent', 'circuit-compiler', 'template', 'create', workspaceTemplateName])
```

Comment:
are we generally use 5 params with `_paq.push`

Reply:
No, i think you can use as many